lib/vuln/subdomain_takeover/engine.py

```diff
@@ -0,0 +1,399 @@
+        ],
+        "sub_takeover_brightcove": [
+            '<p class="bc-gallery-error-code">Error Code: 404</p>'
+        ],
+        "sub_takeover_bigcartel": ["<h1>Oops! We couldn&#8217;t find that page.</h1>"],
+        "sub_takeover_activeCampaign": ['alt="LIGHTTPD - fly light."'],
+        "sub_takeover_campaignmonitor": [
+            'Double check the URL or <a href="mailto:help@createsend.com'
+        ],
+        "sub_takeover_acquia": [
+            "The site you are looking for could not be found.|If you are an Acquia Cloud customer and expect to see your site at this address"
+        ],
+        "sub_takeover_proposify": [
+            'If you need immediate assistance, please contact <a href="mailto:support@proposify.biz"'
+        ],
+        "sub_takeover_simplebooklet": [
+            "We can't find this <a href=\"https://simplebooklet.com"
+        ],
+        "sub_takeover_getResponse": [
+            "With GetResponse Landing Pages, lead generation has never been easier"
+        ],
+        "sub_takeover_vend": ["Looks like you've traveled too far into cyberspace."],
+        "sub_takeover_jetbrains": ["is not a registered InCloud YouTrack."],
+        "sub_takeover_smartling": ["Domain is not configured"],
+        "sub_takeover_pingdom": ["pingdom"],
+        "sub_takeover_tilda": ["Domain has been assigned"],
+        "sub_takeover_surveygizmo": ["data-html-name"],
+        "sub_takeover_mashery": ["Unrecognized domain <strong>"],
+        "sub_takeover_divio": ["Application not responding"],
+        "sub_takeover_airee": ["Ошибка 402. Сервис Айри.рф не оплачен"],
+        "sub_takeover_anima": [
+            "If this is your website and you've just created it, try refreshing in a minute"
+        ],
+        "sub_takeover_hatenablog": ["404 Blog is not found"],
+        "sub_takeover_kinsta": ["No Site For Domain"],
+        "sub_takeover_launchrock": [
+            "It looks like you may have taken a wrong turn somewhere. Don't worry...it happens to all of us."
+        ],
+        "sub_takeover_ngrok": ["Tunnel *.ngrok.io not found"],
+        "sub_takeover_unbounce": ["The requested URL was not found on this server."],
+        "sub_takeover_readme": ["Project doesnt exist... yet!"],
+        "sub_takeover_smartjobboard": [
+            "This job board website is either expired or its domain name is invalid."
+        ],
+        "sub_takeover_strikingly": ["page not found"],
+        "sub_takeover_agilecrm": ["Sorry, this page is no longer available."],
+        "sub_takeover_gemfury": ["404: This page could not be found."],
+        "sub_takeover_getresponse": ["With GetResponse Landing Pages, lead generation has never been easier"],
+        "sub_takeover_juzzhr": ["This account no longer active"],
+        "sub_takeover_landingi": ["The page you are looking for is not found"],
+        "sub_takeover_smugmug": ["{\"text\":\"Page Not Found\""]
```

Comment:
@aman566 have the patterns loaded in a separate wordlist in  lib/payload/wordlists similar to how you implemented dir_scan - also check other subdomain take over patterns defined in nuclei-templates : https://github.com/projectdiscovery/nuclei-templates/blob/master/subdomain-takeover/detect-all-takeovers.yaml

https://github.com/projectdiscovery/nuclei-templates/blob/master/subdomain-takeover/s3-subtakeover.yaml

Reply:
Okay sir,
Done the changes as suggested :)